MapleServer2/PacketHandlers/Game/RequestCubeHandler.cs

```diff
@@ -174,7 +174,7 @@ private static void HandleLoadFurnishingItem(GameSession session, PacketReader p
         private static void HandleBuyPlot(GameSession session, PacketReader packet)
         {
             int groupId = packet.ReadInt();
-            int homeTemplate = packet.ReadInt();
+            int homeTemplate = packet.ReadInt() + 1;
```

Comment:
for what is this ?

Reply:
The client requests home templates from 0~2, the XMLs ids are from 1~3. Should I add an comment or maybe move the +1 to inside the `Home.SetTemplate` function?

Reply:
a comment should be good tho, `// the Templates ids in the XMLs are from 1-3 and the client request 0-2`

Reply:
I changed, at least the +1 isn't on the packet.ReadInt(). It'll stay inside the Home() constructor